zh/grant-permissions-to-remote-storage.md

```diff
@@ -130,6 +130,8 @@ kubectl create secret generic s3-secret --from-literal=access_key=xxx --from-lit
      kubectl annotate sa tidb-controller-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=tidb-admin
      ```
 
+     重启 tidb-controller-manager Pod, 使得 ServiceAccount tidb-controller-manager 生效。
```

Comment:
```suggestion
     重启 tidb-controller-manager Pod，使得 ServiceAccount tidb-controller-manager 生效。
```

Reply:
fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/469d763ab1e94d5c8adea26c36464207d8df77ee